README.md

```diff
@@ -87,11 +93,12 @@ COMMAND="vcf_to_bq \
 docker run -v ~/.config:/root/.config \
   gcr.io/cloud-lifesciences/gcp-variant-transforms \
   --project "${GOOGLE_CLOUD_PROJECT}" \
+  --location "${GOOGLE_CLOUD_LOCATION}" \
   --region "${GOOGLE_CLOUD_REGION}" \
   --temp_location "${TEMP_LOCATION}" \
   "${COMMAND}"
 ```
-`--project`, `--region`, and `--temp_location` are required inputs. You must set all of them, unless your project and region default values
+`--project`, `--location`, `--region`, and `--temp_location` are required inputs. You must set all of them, unless your project and region default values
```

Comment:
Do you think we should still list `--location` as required?

Reply:
Updated to remove listing it as required. also updated same text in setting_region.md.